lib/Backend/arm64/ARM64Encoder.h

```diff
@@ -3500,7 +3500,7 @@ EmitLdrbRegister(
     Arm64RegisterParam Index
     )
 {
-    return EmitLdrStrRegisterCommon(Emitter, Dest, Addr, Index, 0, 0x38600800);
+    return EmitLdrStrRegisterCommon(Emitter, Dest, Addr, Index, IndirScale1, 0x38600800);
 }
```

Comment:
Where is IndirScale1 defined? ARM64Encoder.h is self-contained. I'm leery of introducing dependencies on other chakra components when it isn't necessary.

Reply:
Haha, I see Sandeep suggested you make this change. I'm going to disagree. If you'd like a constant, please defined a new one in ARM64Encoder.h and use it there. Otherwise, I don't think it makes sense to apply this change and create circular dependencies.

Reply:
I like #defines instead of numbers. Define it in ARM64Encoder.h and use.

Reply:
updated to use enums declared in ARM64Encoder.h